sdk/formrecognizer/Azure.AI.FormRecognizer/tests/Infrastructure/DisposableDocumentModel.cs

```diff
@@ -21,18 +20,22 @@ public class DisposableDocumentModel : IAsyncDisposable
         /// Initializes a new instance of the <see cref="DisposableDocumentModel"/> class.
         /// </summary>
         /// <param name="client">The client to use for deleting the model upon disposal.</param>
-        /// <param name="modelId">The identifier of the model to delete upon disposal.</param>
-        private DisposableDocumentModel(DocumentModelAdministrationClient client, string modelId)
+        /// <param name="value">The model to associate with this instance. It will be deleted upon disposal.</param>
+        private DisposableDocumentModel(DocumentModelAdministrationClient client, DocumentModelDetails value)
         {
             _client = client;
-            ModelId = modelId;
+            Value = value;
         }
```

Comment:
Our test helper class `DisposableDocumentModel` is used to create and delete document models for testing. Originally we just stored the model's ID and discarded all the other info the service returns when building a model.

With this PR we're extending the scope of our tests, so the info we were discarding is now necessary. For this reason, we're now storing the whole result in the `Value` property.